pkg/controller/job/job_controller_test.go

```diff
@@ -924,6 +912,8 @@ func checkIndexedJobPods(t *testing.T, control *controller.FakePodControl, wantI
 
 // TestSyncJobLegacyTracking makes sure that a Job is only tracked with
 // finalizers when the job has the annotation.
+
+// Note: With GA of finalizers, do we still need this test?
```

Comment:
@alculquicondor I normally hate to delete tests but do you think we need this anymore?

Reply:
yes, remove it